src/azure-cli/azure/cli/command_modules/backup/commands.py

```diff
@@ -41,6 +41,11 @@ def load_command_table(self, _):
         g.custom_command('encryption update', 'update_encryption')
         g.custom_command('encryption show', 'show_encryption', client_factory=backup_resource_encryption_config_cf)
 
+    with self.command_group('backup vault resource-guard-mapping', backup_custom, client_factory=resource_guard_proxy_cf, exception_handler=backup_exception_handler) as g:
+        g.show_command('update', 'update_resource_guard_mapping')
+        g.show_command('show', 'show_resource_guard_mapping')
+        g.show_command('delete', 'delete_resource_guard_mapping')
```

Comment:
Do we need to add `confirmation=True` for the delete command?

Reply:
Let me check with the PM and come back.